pkg/storaged/btrfs/btrfs-tool.py

```diff
@@ -241,15 +243,34 @@ def cmd_poll(opt_mount):
     sys.stdout.flush()
 
 
+def unshare_mounts():
+    # The os.unshare function is available since Python 3.12, but we
+    # still need to support Python 3.11.
+    libc = ctypes.CDLL(None)
+    libc.unshare.argtypes = [ctypes.c_int]
+    ret = libc.unshare(2**17)
+    if ret != 0:
+        raise Exception("unshare(CLONE_NEWNS) failed")
```

Comment:
`OSError` would be a bit more appropriate

Reply:
Done.